hikari/impl/entity_factory.py

```diff
@@ -437,13 +432,13 @@ def deserialize_audit_log(self, payload: data_binding.JSONObject) -> audit_log_m
                         change_payload["key"]
                     )
 
-                    new_value: typing.Any = change_payload.get("new_value")
-                    old_value: typing.Any = change_payload.get("old_value")
+                    new_value = change_payload.get("new_value")
+                    old_value = change_payload.get("old_value")
                     if value_converter := self._audit_log_entry_converters.get(key):
                         new_value = value_converter(new_value) if new_value is not None else None
                         old_value = value_converter(old_value) if old_value is not None else None
 
-                    elif __debug__ and not isinstance(key, audit_log_models.AuditLogChangeKey):
+                    elif not isinstance(key, audit_log_models.AuditLogChangeKey):
```

Comment:
what's the point of running an instance check during an optimised run if it'll only lead to anything with debug logging which would infer a debug environment?

Reply:
Good point, will document this in the readme and add it back. Will also look for other places to add this :)